adafruit_lis3dh.py

```diff
@@ -74,10 +75,15 @@ def __init__(self):
         self.data_rate = DATARATE_400_HZ
         # High res & BDU enabled.
         self._write_register_byte(REG_CTRL4, 0x88)
-        # DRDY on INT1.
-        self._write_register_byte(REG_CTRL3, 0x10)
         # Enable ADCs.
         self._write_register_byte(REG_TEMPCFG, 0x80)
+        # Initialise interrupt pins
+        self._int1 = int1
```

Comment:
Passing in the DigitalInOut pin means it could be another compatible object instead. The only requirement we pose is that it provides `value`, `direction` and `pull`. So, this could actually be available through a GPIO expander. Taking the board pin restricts our use to directly connected pins.

Reply:
Got it. I though you might want to duck-type something.